donut/modules/editor/templates/created_list.html

```diff
@@ -1,26 +1,28 @@
 {% extends "layout.html" %}
 {% block page %}
+<head>
 <script src="https://cdn.rawgit.com/showdownjs/showdown/1.0.1/dist/showdown.min.js"></script>
 <script src="{{ url_for('uploads.static', filename='script.js', _external = True)}}" type="text/javascript"></script>
+</head>
```

Comment:
`<head>` tag should not go inside `<body>`, which is where `{% block page %}` is inserted. Put scripts in `{% block scripts %}` instead.